src/main/java/Winston.java

```diff
@@ -0,0 +1,41 @@
+import java.util.Scanner;
+
+import winston.Command;
+import winston.Parser;
+import winston.Storage;
+import winston.TaskList;
+import winston.TerminateCommand;
+import winston.Ui;
+
+/**
+ * A class that creates your personal assistant Winston
+ */
+public class Winston {
+    private Storage storage;
+    private TaskList taskList;
+    private Ui ui;
+    private Parser parser;
+
+    /**
+     * Constructor for class Winston
+     */
+    public Winston() {
+        this.storage = new Storage();
+        this.taskList = new TaskList(storage.read());
+        taskList.setStorage(this.storage);
+        this.parser = new Parser(taskList);
+    }
+    
+    public static void main(String[] args) {
+        boolean isExit = false;
+        Winston winston1 = new Winston();
```

Comment:
Try not to use numbers for variables. Perhaps naming a variable with something more meaningful will make it clearer